### PR TITLE
DOCS: Add FAQ articles for new Magento 2 plugin

### PR DESCRIPTION
**Pull request by Vinod-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/integrations/plugins/magento2/faq/payment-fee-surcharges.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Adding a payment fee or [Surcharge](/faq/getting-started/glossary/#surcharge) is no longer supported within the plugin in Magento 2.
+
+However, an external package of [Fooman](https://store.fooman.co.nz/extensions/magento2) is available. This allows you to still add a payment fee or surcharge within the desired payment method.
+
+Refunding from the Magento 2 backend is disabled when the order has a Fooman Surcharge. It is still possible to refund those orders through [MultiSafepay Control](https://merchant.multisafepay.com)
+
+_Do mind that because an external module has to be installed, our Integration Team's help will be limited_.
+
+Of course, we do our best to support and assist as best as possible, but we investigate or reproduce on the core of a plugin. Therefore, we do not guarantee a perfect compatibility when installing an external package.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/payment-fee-surcharges.md:18]:
```suggestion
Of course, we do our best to support and assist as best as possible, but we investigate or reproduce on the core of a plugin. Therefore, we do not guarantee perfect compatibility when installing an external package.
```

File: content/integrations/plugins/magento2/faq/payment-links-in-order-confirmation-for-backend-orders.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+layout: "faqdetail"
+summary: "As of version 1.7.0, we have added a feature to include the payment link in the order confirmation mail. This feature is customizable and can be changed to your liking."
+read_more: "."
+aliases:
+    - /magento2/faq/payment-links-in-order-confirmation-for-backend-orders
+---
+
+As of version 1.7.0, we have added a feature to include the payment link in the order confirmation mail. This feature is customizable and can be changed to your liking. This feature can be implemented by the following steps:
+
+1. Login into your Magento backend. Check how we do it [here](/integrations/plugins/magento2/old/) in step 5.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/payment-links-in-order-confirmation-for-backend-orders.md:14]:
```suggestion
1. log in to your Magento backend. Check how we do it [here](/integrations/plugins/magento2/old/) in step 5. 
```

File: content/integrations/plugins/magento2/faq/payment-links-in-order-confirmation-for-backend-orders.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+1. Login into your Magento backend. Check how we do it [here](/integrations/plugins/magento2/old/) in step 5. 
+2. Go to _Marketing_ -> _Email Templates_
+3. Add a template (import from _new order_)
+4. Add this sample code the template
+</br>
+`{{depend order.getPayment().getAdditionalInformation('payment_link')}}`
+`<a href="{{var order.getPayment().getAdditionalInformation('payment_link')}}">Pay now with {{var order.getPayment().getAdditionalInformation('method_title')}}</a>`
+`{{/depend}}`\
+</br>\
+The email template syntax is different for Magento 2.3.4+. For these versions you can add this sample code instead
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/payment-links-in-order-confirmation-for-backend-orders.md:23]:
```suggestion
The email template syntax is different for Magento 2.3.4+. For these versions, you can add this sample code instead
```

File: content/integrations/plugins/magento2/faq/refunding-magento2.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title : "Can I refund orders?"
+meta_title: "Magento 2 plugin FAQ - Refund - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+layout: "faqdetail"
+read_more: "."
+---
+Yes, it is possible to refund orders or credit memo from within the Magento 2 backend.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/refunding-magento2.md:8]:
```suggestion
Yes, it is possible to refund orders or credit memos from within the Magento 2 backend.  
```

File: content/integrations/plugins/magento2/faq/refunding-magento2.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title : "Can I refund orders?"
+meta_title: "Magento 2 plugin FAQ - Refund - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+layout: "faqdetail"
+read_more: "."
+---
+Yes, it is possible to refund orders or credit memo from within the Magento 2 backend.  
+
+In the Magento backoffice, go to the Invoices tab, click on the invoice that was created by MultiSafepay and then click Credit Memo. Now you see 2 refund buttons:
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/refunding-magento2.md:10]:
```suggestion
In the Magento backoffice, go to the _Invoices_ tab, click on the invoice that was created by MultiSafepay, and then click _Credit Memo_. Now you see 2 refund buttons:
```

File: content/integrations/plugins/magento2/faq/refunding-magento2.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+* Offline refund: refund request will not be sent to MultiSafepay
+* Refund: a refund request will be sent to MultiSafepay.
+
+You can also refund from your [MultiSafepay Control](https://merchant.multisafepay.com)
+
+_Note: Refunding from the Magento 2 backend is disabled when the order has a Fooman Surcharge. It is still possible to refund those orders through [MultiSafepay Control](https://merchant.multisafepay.com)_
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/refunding-magento2.md:16]: Add a line that refers to our new Refunds (/faq/refunds/) sections for more questions on refunds

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]
Please revise

File: content/integrations/plugins/magento2/faq/how-can-i-generate-a-payment-link.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+title : "How can I generate a payment link in the backend of Magento 2?"
+meta_title: "Magento 2 payment link - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+layout: "faqdetail"
+read_more: "."
+---
+
+From version 2.0 and higher, payment links are automatically generated for every order with a MultiSafepay payment method. Please follow these instructions:
```
[the remark this answers is not in the document]
Comment (Vinod-MultiSafepay) [reply content/integrations/plugins/magento2/faq/how-can-i-generate-a-payment-link.md]: A MultiSafepay payment method is a payment method that has been made available by MultiSafepay. I put in 'MultiSafepay'. Maybe that is already clear for the user that is reading this and we can just say 'payment method' instead?

File: content/integrations/plugins/magento2/faq/does-the-plugin-support-Magento-2-Commerce.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Does the plugin support Magento 2 Commerce?"
+meta_title: "Magento 2 Commerce support - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+read_more: "."
+---
+
+If you encounter issues or require assistance please contact [integration@multisafepay.com](mailto:integration@multisafepay.com)
```
[the remark this answers is not in the document]
Comment (Vinod-MultiSafepay) [reply content/integrations/plugins/magento2/faq/does-the-plugin-support-Magento-2-Commerce.md:8]: This is how it also has been for the old plugin faq, do you suggest a better way?

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**